### ISSUE TITLE
p-selectbutton has conflicting aria attributes

### ISSUE THREAD

**Issue by Cue-Ai-NGinEar** [primefaces/primeng#15227]
### Describe the bug

Analysing the p-selectbutton component using axe-core (or other plug-ins such as wave) highlights that the p-selecttbutton component has both the `aria-pressed` and `aria-checked` attributes set. This essentially means it has characteristics of both a radio button and a traditional button.  
![image](https://github.com/primefaces/primeng/assets/151757425/93a7e832-c5f9-4c12-9908-2e0bb5ffd231)

### Environment

v17.12.0

### Angular version

17.3.2

### PrimeNG version

17.12.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.3.0

### Steps to reproduce the behavior

Scan the p-selectbutton component in situ.

### Expected behavior

The design pattern for the p-selectbutton component shouldn't flag violations against WCAG due to having both the `aria-checked` and `aria-pressed` attributes.

**Comment by mehmetcetin01140** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
